pkg/daemon/rpm-ostree.go

```diff
@@ -124,6 +124,10 @@ func (r *RpmOstreeClient) RunPivot(osImageURL string) error {
 	defer close(journalStopCh)
 	go followPivotJournalLogs(journalStopCh)
 
+	if err := os.Remove("/run/pivot/reboot-needed"); err != nil && !os.IsNotExist(err) {
```

Comment:
Also this one could really use a comment like:

`// This is written by code injected by the MCS, but we always want the MCD to be in control of reboots`